asciidoc-confluence-publisher-cli/pom.xml

```diff
@@ -62,6 +62,11 @@
             <artifactId>hamcrest-all</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Please add the `httpclient` dependency after the `jackson-databind` dependency (we first list our own dependencies, than all non-test-scoped dependencies, then all test-scoped dependencies, alphabetically sorted by the group-id).